rclcpp/include/rclcpp/executor.hpp

```diff
@@ -913,14 +800,14 @@ class Executor
 private:
   RCLCPP_DISABLE_COPY(Executor);
 
-  std::vector<std::weak_ptr<rclcpp::node::Node>> weak_nodes_;
-  typedef std::list<void*> SubscriberHandles;
+  std::vector<std::weak_ptr<rclcpp::node::Node> > weak_nodes_;
```

Comment:
I would not add a space here. The lack of a space here will not be a problem on any C++ compiler we will support.